tools/theme/src/Theme.ts

```diff
@@ -163,14 +180,34 @@ export class Theme extends HTMLElement implements ThemeKindProvider {
             this._provideContext();
         } else if (attrName === 'theme') {
             this.theme = value as SystemVariant;
-            window.__swc.warn(
-                this,
-                'property theme in <sp-theme> has been deprecated. Please use system instead like this <sp-theme system="spectrum"/>',
-                'https://opensource.adobe.com/spectrum-web-components/tools/themes/#deprecation',
-                { level: 'deprecation' }
-            );
+            if (window.__swc.DEBUG) {
```

Comment:
Thanks for wrapping this warn in this gate, it is required to ensure that Dev Mode does not get shipped to production.